model/cluster.go

```diff
@@ -193,6 +194,20 @@ type EKSClusterModel struct {
 	DefaultUser        bool
 	ClusterRoleId      string
 	NodeInstanceRoleId string
+
+	LogTypes EKSLogTypes `sql:"type:json"`
+}
+
+type EKSLogTypes []string
+
+// Value implements the driver.Valuer interface
+func (elt EKSLogTypes) Value() (driver.Value, error) {
+	return json.Marshal(elt)
+}
+
+// Scan implements the sql.Scanner interface
+func (elt *EKSLogTypes) Scan(src interface{}) error {
+	return json.Unmarshal(src.([]byte), &elt)
```

Comment:
```suggestion
	return json.Unmarshal(src.([]byte), elt)
```
`elt` is already a pointer

Reply:
thanks fixed!